python/setup.py

```diff
@@ -0,0 +1,12 @@
+import setuptools
+
+setuptools.setup(
+    name="hyperspace",
+    version="0.1.0",
+    description="Python bindings for hyperspace.",
+    url="https://github.com/microsoft/hyperspace",
+    packages=setuptools.find_packages(),
+    classifiers=[
+    ],
```

Comment:
Why the linebreak?

Reply:
removed file. thanks. Now py files should be in jar